src/components/TableMatrix/TableHeader/TableHeader.tsx

```diff
@@ -0,0 +1,29 @@
+import styles from '../TableHeader/TableHeader.module.scss';
+
+const topCells = (value: string | number) => {
+  return (
+    <th className={styles.headerItem} key={value}>
+      {value}
+    </th>
+  );
+};
+
+const TopUnit = (columns: number) => {
+  const banner = [topCells('№')];
+  for (let i = 0; i < columns; i++) {
+    banner.push(topCells(i + 1));
```

Comment:
it is better to write 
_**banner = [...banner, newItem]**_ instead of _**push**_ because of performance